scvi/dataset/benchmark_hyperparameters.py

```diff
@@ -0,0 +1,19 @@
+from . import BrainLargeDataset
+
+
+def benchmark_hyperparameters(gene_dataset):
+    """
+    Possibly specify hyper parameters according to Table2 in "Bayesian Inference for a Generative Model of
+    transcriptome Profiles from Single-cell RNA Sequencing"
+    :param gene_dataset:
+    :return:
+    """
+    hyperparameters = dict()
+    if isinstance(gene_dataset, BrainLargeDataset):
+        if gene_dataset.nb_genes > 10000 and gene_dataset.nb_genes < 15000:
+            hyperparameters["n_layers"] = 2
+        elif gene_dataset.nb_genes > 15000:
+            hyperparameters["n_layers"] = 3
+            hyperparameters["n_hidden"] = 256
+
```

Comment:
@Edouard360 Better to put this code in `benchmark.py` rather than the `dataset` directory.

Reply:
yes indeed, just changed it. Otherwise should I keep it as a function ?

Reply:
Yes, I think it's fine. I appreciate you fixing issue #64 . 

Eventually we may want to define a `Benchmark` class -- each instance has a vae, a dataset, a name string (e.g., `brain_large`), and some hyperparameters. To make this approach work, we'd need to stop downloading and preprocessing the dataset whenever it's `__init__` method is called.